dev/com.ibm.websphere.appserver.features/visibility/public/springBoot-4.0/io.openliberty.springBoot-4.0.feature

```diff
@@ -6,7 +6,7 @@ IBM-ShortName: springBoot-4.0
 IBM-Process-Types: server
 Subsystem-Name: Spring Boot Support 4.0
 -features=io.openliberty.springBootHandler-4.0, \
-  com.ibm.websphere.appserver.eeCompatible-10.0; ibm.tolerates:="11.0"
+  com.ibm.websphere.appserver.eeCompatible-11.0
```

Comment:
EE10Features change that was done in io.openliberty.jakartaee10.internal_fat will also need to be undone as well.